### DISCUSSION TITLE
tegra-libraries_32.6.1.bb recipe should it be removed

### DISCUSSION THREAD

**Discussion by MikeLoder** [OE4T/meta-tegra#985] [General]
I noticed in the Dunfell branch there is a tegra-libraries_32.6.1.bb recipe its still marked at the lower version and nothing depends on it. I think this file should have its version up-reved, or the recipe removed. Just want to discuss to see if I am missing why this was done.

Thanks

**Comment by madisongh** [comment]
Yeah, it should probably be removed.  I had it in there to help with transitions from earlier branches/L4T versions before we switched to the more granular recipes.

**Reply by MikeLoder** [reply]
Thanks for the confirmation, i will submit a PR to remove it
